src/components/Navbar.tsx

```diff
@@ -12,15 +12,17 @@ const Navbar = ({
   showExploreButton = false,
   showSocialIcons = false,
 }: NavRouteProps) => (
-  <RBNavbar className="pt-4 justify-content-center">
-    <Nav>{showBrandIcon ? <NavbarBrand /> : <BrandPlaceholder />}</Nav>
-    <Nav className="align-items-center align-self-center">
-      {showExploreButton ? <Nav.Link to="/explore" as={Link}>Explore</Nav.Link> : <></>}
-      &nbsp;
-      {showSocialIcons ? <SocialIcons /> : <></>}
-      &nbsp;
-      {showAccount ? <AccountNavbar /> : <></>}
-    </Nav>
+  <RBNavbar className="pt-4">
+    <Container>
+      <Nav>{showBrandIcon ? <NavbarBrand /> : <BrandPlaceholder />}</Nav>
+      <Nav className="align-items-center align-self-center">
+        {showExploreButton ? <Nav.Link to="/explore" as={Link}>Explore</Nav.Link> : <></>}
+        &nbsp;
+        {showSocialIcons ? <SocialIcons /> : <></>}
+        &nbsp;
+        {showAccount ? <AccountNavbar /> : <></>}
```

Comment:
Not a big deal, but from what I recall, this is the most conventional way of conditionally showing a component in React:

```suggestion
        {showExploreButton && <Nav.Link to="/explore" as={Link}>Explore</Nav.Link>}
        &nbsp;
        {showSocialIcons && <SocialIcons />}
        &nbsp;
        {showAccount && <AccountNavbar />}
```